cluster/common.sh

```diff
@@ -300,7 +300,6 @@ function find-tar() {
   local -r tarball=$1
   locations=(
     "${KUBE_ROOT}/server/${tarball}"
-    "${KUBE_ROOT}/kubernetes/server/${tarball}"
```

Comment:
Not sure why this was added, but I don't think we need this line. @pjh

Reply:
Did we add this? If so I don't remember when or why.

Reply:
`git blame` said you added that :P

I've never had a `kubernetes` directory inside KUBE_ROOT. I think we'll be fine.